src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+document.addEventListener('DOMContentLoaded', () => {
+  const theadRow = document.querySelector('table thead tr');
+  const theadCells = theadRow.querySelectorAll('th');
+  const clonedTheadCell = theadCells[1].cloneNode(true);
+
+  theadRow.insertBefore(clonedTheadCell, theadCells[4]);
```

Comment:
Similar to the previous comment, ensure that `theadCells[4]` exists before attempting to insert the cloned cell. This will prevent errors if the table has fewer than 5 columns.